pkgs/top-level/all-packages.nix

```diff
@@ -21452,12 +21452,17 @@ in
     inherit name src;
     depsBuildBuild = [ buildPackages.stdenv.cc ]
       ++ lib.optionals (lib.versionAtLeast version "4.16") [ buildPackages.bison buildPackages.flex ];
+    prePatch = ''
```

Comment:
```suggestion
    postPatch = ''
```

Reply:
You should say why you propose this change.

Here I stopped a few seconds and thought. I can see *one* use case. If the derivation is overriden with a patch that adds files under `scripts/` that requires to be patched, the `prePatch` step wouldn't have patched their shebangs.

Still, I don't know why **you** want this change to be made.

Reply:
Changing files in prePatch breaks upstream patches.

I am to lazy to write that under every suggestion.